tests/searchcommands/test_searchcommands_app.py

```diff
@@ -86,16 +86,20 @@ def __init__(self, path):
         splunk_cmd = path + '.splunk_cmd'
 
         try:
-            with io.open(splunk_cmd, 'rb') as f:
+            with io.open(splunk_cmd, 'r') as f:
```

Comment:
why the change from 'rb' to 'r'?

Reply:
This is to fix a bug that manifests in Python 3. The line below attempts to call `.readline` on a binary-opened filestream, which conceptually you shouldn't be able to do. Python 2 allows you to do it because it has a very loose idea of what bytes and strings are, but it crashes in Python 3.

Reply:
Makes sense, thanks.